woocommerce/woocommerce-gateway-stripe
# Use JSON.parse instead of the deprecated jQuery.parseJSON

Make sure #1392 is merged before this PR to bring the composer package updates to `master`.

# Changes proposed in this Pull Request:

Fixes #1389

Remove the use of the deprecated `jQuery.parseJSON()` function, and instead use the builtin `JSON.parse()`.

# Testing instructions

Requires WooCommerce Subscriptions.

- Create a subscription.
- Process an early renewal through the account page—not through the admin dashboard.

Make sure there are no JS errors.

While using WP 5.6 you might see a jQuery Migrate warning about deprecated code. So long as the warning isn't about a `jQuery.parseJSON()` call in `stripe.js` or any other JS script in the Stripe gateway, this change is good.

-------------------
